app/task_routes.py

```diff
@@ -0,0 +1,134 @@
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, abort, request
+import datetime
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix = "/tasks")
+
+###### Create Route ######
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if not request_body.get("title") or not request_body.get("description"):
```

Comment:
Like we validated models, could we also make a general function that validates request bodies? Here's an example:
```python
def validate_request_body(request_body, keys):
    for key in keys:
        if not request_body.get(key): 
            abort(make_response({
                'Invalid Data': f'missing key: {key}'
            }, 400))

    return True
```
We can pass in the `request_body` and a list of strings that are `keys` and then check to see if those keys are present.